.github/workflows/tests.yml

```diff
@@ -121,6 +121,10 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Maybe mark for benchmarking
+        if: always() # && github.ref == 'refs/heads/main' && github.repository == 'coiled/coiled-runtime'
```

Comment:
That was my intention, yes. Do these conditions seem reasonable to you?

Reply:
I think so? Looking at the [docs for `github.ref`](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) I see

> This is only set if a branch or tag is available for the event type

So I guess there are some cases where it might not work as expected (in particular I'm wondering about the cron job we have running on `main`). Regardless, I think this we should give this a try and can always update if needed

Reply:
Good point. The [scheduling](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule) functionality only works on the default branch, so it seems like `github.ref` should be set there. But I suppose we will find out.